hunting-buddy.lic

```diff
@@ -207,11 +207,17 @@ class HuntingBuddy
       @exit = [escort_info['area'], 'exit']
     else
       @exit = nil
-      find_empty_room(rooms, waiting_room,
-                      lambda do
+      return find_empty_room(rooms, waiting_room,
+                      lambda do |search_attempt|
+                        # Skip room if has more people than your max limit
                         return false if DRRoom.pcs.size > @hunting_buddies_max
+                        # Skip if one of your nemesis is in the room
                         return false if (DRRoom.pcs & UserVars.hunting_nemesis).any?
+                        # Continue if one of your friends is in the room
                         return true if (DRRoom.pcs & UserVars.friends).any?
+                        # Continue if this is not the first search attempt and character will settle for "good enough" room
+                        return true if search_attempt > 1 && !@settings.hunting_room_strict_empty_or_buddies
```

Comment:
It would be helpful for me and the way I handle getting my attention if something non-standard is going on, if this echo'd something when it makes this choice (like the mana one does) so that I could then know I should say something to the random people i'm now hunting with.

Reply:
Per feedback, I've removed this aspect from the pull request to relax criteria to hunt with people not explicitly listed in your buddies list.